examples/tools.rst

```diff
@@ -14,3 +14,4 @@ Conan recipe tools examples
    tools/scm/git/capture_scm/git_capture_scm
    tools/microsoft/msbuild
    tools/system/package_manager
+   tools/system/consuming_system_packages
```

Comment:
These need to be put into the same ``tools/system`` examples directory, not a first level entry for each one.

The entry point can summarize and redirect to the different examples below it

Reply:
Do you mean only extending the file `package_manager.rst` with this new content, instead of adding an entire new?

This new example/how-to is under the same examples tools/system folder as the other example.

Reply:
I mean, every other item in this section is a sub-section, see https://docs.conan.io/2/examples/tools.html, containing further articles/examples. The missing part is the equivalent to https://docs.conan.io/2/examples/tools/google/bazel.html, but for "system-package-manager"

Reply:
Restructured on the commit [6f50b49](https://github.com/conan-io/docs/pull/3980/commits/6f50b49f3f99828bb2dcf2e3269a51958c1ac25a)